### ISSUE TITLE
Germany error for March 13 (date is set to March 11)

### ISSUE THREAD

**Issue by jmoeckel** [CSSEGISandData/COVID-19#963]
The "Last Update" value for Germany on March 13 is set to March 11 - but the numbers do not fit.

Germany | 2020-03-11T19:13:17 | 1908 | 3 | 25 from file 03-11-2020.csv
Germany | 2020-03-12T09:53:06 | 2078 | 3 | 25 from file 03-12-2020.csv
Germany | 2020-03-11T20:00:00 | 3675 | 7 | 46 from file 03-13-2020.csv

Most likely, the "Last Update" value for Germany on 03-13-2020.csv must be changed to 2020-03-13T20:00:00.

**Comment by JiPiBi** [comment]
Yes,  there are some slight errors like that or more important on values , but I never saw a correction in the daily reports .....
Could be the first one  :-)

**Comment by jmoeckel** [comment]
yeah, hope so :-). 

This error in particular produces negative daily numbers while using [https://github.com/AaronWard/covid-19-analysis](https://github.com/AaronWard/covid-19-analysis).

**Comment by JiPiBi** [comment]
Another solution if you use tools like pandas , or others , is after downloading to fix the values. 

  If you are searching for in the issues , youll find that some people made that work  , code or even code + data on their own repo.
Up to you.
